raveandroid/src/main/java/com/flutterwave/raveandroid/Encrypt.java

```diff
@@ -0,0 +1,29 @@
+package com.flutterwave.raveandroid;
+
+import javax.inject.Inject;
+
+public class Encrypt {
```

Comment:
@mikelis135 Delete this class. It looks unused

Reply:
done